docs/_partner_api/endpoints.md

```diff
@@ -1,13 +1,125 @@
 ---
-title: Endpoints
-order: 6
+title: Partner API
+order: 3
 layout: post-toc
 redirect_from: /partner_api/
```

Comment:
If this replaces partner_api/endpoints we'll need to add this to redirect_from to cover the permalink